.github/workflows/nightly-release.yaml

```diff
@@ -33,10 +33,9 @@ jobs:
           sudo apt update -qq
           sudo apt install -y -qq build-essential git sqlite libtag1-dev ffmpeg mpv zlib1g-dev
       - name: Lint
-        uses: golangci/golangci-lint-action@v2
+        uses: golangci/golangci-lint-action@v3
         with:
-          version: v1.50.1
-          skip-go-installation: true
+          version: v1.52.2
```

Comment:
why do we get rid if skip-go-installation? iirc it was true because we have the setup go action above

Reply:
The option has been removed in the new version of `golangci-lint-action`, see: https://github.com/golangci/golangci-lint-action#compatibility:

> v3.0.0+ requires explicit setup-go installation step prior to using this action: uses: actions/setup-go@v3. The skip-go-installation option has been removed.